tool/public/index.html

```diff
@@ -27,9 +31,9 @@
 			<textarea class="form-control h-100 min-h-0 lh-1 font-monospace" id="ipt-text"></textarea>
 		</div>
 		<div class="col-6 h-100 d-flex flex-column">
-			<div class="input-group mb-3">
-				<button type="button" class="btn btn-primary btn-sm" id="btn-convert">Convert</button>
-				<button type="button" class="btn btn-primary btn-sm" id="btn-copy">Copy</button>
+			<div class="mb-3">
+				<button type="button" class="btn btn-primary btn-sm me-1 w-80p" id="btn-convert">Convert</button>
```

Comment:
`me-1`?

Reply:
[Bootstrap](https://getbootstrap.com/docs/5.1/utilities/spacing/#notation) class in the same group as `mb-3`. It means 'margin - end (right) - 1'.

Just adds a bit of horizontal spacing between the buttons (I removed the input-group because it looked kinda weird with the extra-wide buttons).

Reply:
you know, I totally forgot this is _actual_ bootstrap, and not my bastardized semi-clone. Carry on!